### DISCUSSION TITLE
Pb with EdegTX 2.6 on Horus X12S . The internal module X12S have FLYSKY , Horus = FRSKY

### DISCUSSION THREAD

**Discussion by captflame2121** [EdgeTX/edgetx#1591] [Ideas]
Hi , 
I will want to use my internal modul on my X12S Horus FRSKY with EdegTX 2.6 .
I have doing a second flash for be sure . But If you want select internal module you have FLYSKY .
And I will want bind my receiver X8R PRO but in the internal module you don't have FRSKY.  

It's one error because with 2.5 internal module is good and FRSKY .

Bests regards

**Comment by pfeerick** [comment]
What does it say under SYS -> Hardware (tab) -> Internal module (near the bottom)? It should say XJT. Perhaps it's set to MULTI on yours or you are looking at the External RF setting? As FLYSKY should *not* be an option on the X12S internal RF, and does not appear to be the case when I check on the simulator.

**Comment by captflame2121** [comment]
Hi Peter, Sorry for my English.
I use RX8R Frsky with OpenTX with internal RF with my Horus X12.

Yes when you use the simulator you have XJT for Horus X12 to EdgeTX 2.5 companion

Look the picture when I flash my Horus X12S to EdegTX 2.6
I have FLYSKY to choice in internal RF . One Choice
[cid:830ebc34-5c60-4f9f-ba02-80a5dae1f96f]

If I reflash with EdegTX 2.5 for Horus X12S you have FRsky in internal RF.

It's in the firmware that is a problem ?  No

Best Regards

Thanks

FLORENT



________________________________
De : Peter Feerick ***@***.***>
Envoyé : lundi 14 février 2022 01:43
À : EdgeTX/edgetx ***@***.***>
Cc : captflame2121 ***@***.***>; Author ***@***.***>
Objet : Re: [EdgeTX/edgetx] Pb with EdegTX 2.6 on Horus X12S . The internal module X12S have FLYSKY , Horus = FRSKY (Discussion #1591)


What does it say under SYS -> Hardware (tab) -> Internal module (near the bottom)? It should say XJT. Perhaps it's set to MULTI on yours or you are looking at the External RF setting? As FLYSKY should not be an option on the X12S internal RF, and does not appear to be the case when I check on the simulator.

—
Reply to this email directly, view it on GitHub<https://github.com/EdgeTX/edgetx/discussions/1591#discussioncomment-2169786>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AI7N2UY5V4CKMLLVUMFJB33U3BF3DANCNFSM5OJHE4TA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by pfeerick** [reply]
Also for the 2.6 companion, so I am note sure what is going on there. You'll have to add that screenshot here on github as attached images don't come through. 

Did you check this?
> What does it say under SYS -> Hardware (tab) -> Internal module (near the bottom)? 

It is this setting that informs what should be used in the model settings, so if that is somehow set to FLYSKY, that is the problem. 

Can you zip up your MODELS and RADIO folders so I can have a look at what is going on with the settings?

**Comment by captflame2121** [comment]
Hi Peter,

  1.  I can put the picture on Github
  2.  When I say SYS  I want to say  the system protocol radio , I don't use external module
  3.  my picture show internal module 's choice    .

      3. If I FLASH my Horus x12 with EdgeTX 2.5 to the internal module you have FRSKY
      4.  I change nothing
      5. If I FLASH my Horus x12 with EdgeTX 2.6 to the internal module you have FLYSKY ( Flysky is for Nirvana Radio not Horus)

The problem is not with my MODELS and RADIO folders because when I flash in 2.5 EdegTX  it's good I have FRSKY to internal Module .
For me it's the firmware EdegTX that write internal module ?

This is the picture  backup x12s on my PC
Defaut mod intern  is XJT  . I have doing a synchronisation pc and radio. I have transfert after to y radio .

[cid:7a79c638-4f89-445d-b92c-51b1f8602be2]


I turn off my radio x12, I start radio and to internal module I don't have XJT FRSKY , I have Flysky




Thanks and best regards



________________________________
De : Peter Feerick ***@***.***>
Envoyé : mardi 15 février 2022 00:40
À : EdgeTX/edgetx ***@***.***>
Cc : captflame2121 ***@***.***>; Author ***@***.***>
Objet : Re: [EdgeTX/edgetx] Pb with EdegTX 2.6 on Horus X12S . The internal module X12S have FLYSKY , Horus = FRSKY (Discussion #1591)


Also for the 2.6 companion, so I am note sure what is going on there. You'll have to add that screenshot here on github as attached images don't come through.

Did you check this?

What does it say under SYS -> Hardware (tab) -> Internal module (near the bottom)?

It is this setting that informs what should be used in the model settings, so if that is somehow set to FLYSKY, that is the problem.

Can you zip up your MODELS and RADIO folders so I can have a look at what is going on with the settings?

—
Reply to this email directly, view it on GitHub<https://github.com/EdgeTX/edgetx/discussions/1591#discussioncomment-2177057>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AI7N2U4CQAH32X2ESJKXDKLU3GHFJANCNFSM5OJHE4TA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by pfeerick** [reply]
Hi Florent,

The setting I was asking you to check *is* for the internal RF module, not an external one (as that is configured on a model by model basis, not in the radio hardware settings) - since as of 2.6 the setting for the internal module can be changed!!!

> What does it say under SYS -> Hardware (tab) -> **Internal** module (near the bottom)?

Check on the radio, not on Companion, as it may be a bug in Companion which is setting the wrong internal module setting.

This is also the reason I would like to see what is going on in the `RADIO/radio.yml` file - as I want to see what the internal module configuration setting is currently set to. 

You will need to post images directly on github, as images added/attached via email are not shown.

**Comment by captflame2121** [comment]
Hi Peter,
I had take new SD card with the folder Horus EdgeTX 2.6  and now it's good .  I have now FRSKY to my internal module ( with d8 d16 and L12).

But the bind with my rx RX8R FRSKY is not good .  I think the firmware betwen the rx and the internal module X12S is different .

The problem now is that I don't know how to flash internal module Horus X12  and a RX Frsky   to EdegTX 2.6 .
Do you know if there are firmware EdegTX 2.6 for write to the internal module X12 and RX Frsky  . Because the rx and the tx must have the same firmware .

Before to Open tx I could change the firmware internal module X12 ( Accst , Access ...)  and I thought the same firmware for RX but Now I don't know where is it or exist it ?

Thanks Peter and sorry for my problems .

best regards

Florent


________________________________
De : Peter Feerick ***@***.***>
Envoyé : mercredi 16 février 2022 11:59
À : EdgeTX/edgetx ***@***.***>
Cc : captflame2121 ***@***.***>; Author ***@***.***>
Objet : Re: [EdgeTX/edgetx] Pb with EdegTX 2.6 on Horus X12S . The internal module X12S have FLYSKY , Horus = FRSKY (Discussion #1591)


Hi Florent,

The setting I was asking you to check is for the internal RF module, not an external one (as that is configured on a model by model basis, not in the radio hardware settings) - since as of 2.6 the setting for the internal module can be changed!!!

What does it say under SYS -> Hardware (tab) -> Internal module (near the bottom)?

Check on the radio, not on Companion, as it may be a bug in Companion which is setting the wrong internal module setting.

This is also the reason I would like to see what is going on in the RADIO/radio.yml file - as I want to see what the internal module configuration setting is currently set to.

You will need to post images directly on github, as images added/attached via email are not shown.

—
Reply to this email directly, view it on GitHub<https://github.com/EdgeTX/edgetx/discussions/1591#discussioncomment-2187724>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AI7N2UZEXQERZWJMTPCNYBLU3N7STANCNFSM5OJHE4TA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by captflame2121** [comment]
Hi Peter I'm sorry,

I think that my problem where I can't bind  my RX and my Horus X12s is the SD pack Horus that EdegTX give  to  https://github.com/EdgeTX/edgetx-sdcard/releases

The SD Pack Horus is for Horus X10 and X10S because module internal is XJT  and for the Horus X12S  it's IXJT   for internal module.

And with my new SD Card and the pack horus  https://github.com/EdgeTX/edgetx-sdcard/releases       I have now XJT to the internal module but for Horus X12S it's not XJT , this  IXJT

So I can't bind RX Frsky with my horus X12 with the SD Card pack gived .


Grrrrrrrrrrr



Bests Regards



________________________________
De : Peter Feerick ***@***.***>
Envoyé : mercredi 16 février 2022 11:59
À : EdgeTX/edgetx ***@***.***>
Cc : captflame2121 ***@***.***>; Author ***@***.***>
Objet : Re: [EdgeTX/edgetx] Pb with EdegTX 2.6 on Horus X12S . The internal module X12S have FLYSKY , Horus = FRSKY (Discussion #1591)


Hi Florent,

The setting I was asking you to check is for the internal RF module, not an external one (as that is configured on a model by model basis, not in the radio hardware settings) - since as of 2.6 the setting for the internal module can be changed!!!

What does it say under SYS -> Hardware (tab) -> Internal module (near the bottom)?

Check on the radio, not on Companion, as it may be a bug in Companion which is setting the wrong internal module setting.

This is also the reason I would like to see what is going on in the RADIO/radio.yml file - as I want to see what the internal module configuration setting is currently set to.

You will need to post images directly on github, as images added/attached via email are not shown.

—
Reply to this email directly, view it on GitHub<https://github.com/EdgeTX/edgetx/discussions/1591#discussioncomment-2187724>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AI7N2UZEXQERZWJMTPCNYBLU3N7STANCNFSM5OJHE4TA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
